about-us.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About-us</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,900&family=Poppins:wght@400;700&family=Raleway:wght@300;900&display=swap" rel="stylesheet">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="top">
+                <a href=""><img src="images/facebook.png" alt="facebook"></a>
+                <a href=""><img src="images/twitter.png" alt="twitter"></a>
+                <ul>
+                    <li>English</li>
+                    <li>MyPage</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="bottom">
+                <a href="index.html"><img src="images/Logopit_1688405566257.png" alt="logo"></a> 
+                <div class="nav-bottom-links">
+                    <ul>
+                        <li><a class="active" href="about-us.html" >About</a></li>
+                        <li><a href="#">Blog</a></li>
+                        <li><a href="#">Courses</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                        <li><button class="btn red">CC Campaign</button></li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
```

Comment:
### HOME AND ABOUT PAGE

- [ ] As mentioned by the previous reviewer :point_up_2: please improve your design by aligning each nav to improve the spacing as seen in the main design improve your design as much as we can but still your navbar is not aligned properly so kindly fix this issue :+1:  check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-07 01-18-15](https://github.com/ethred/Capstone/assets/45226848/8de3ef7d-c38a-4662-bbc0-118b5afb3d60)

**Behance Design**
![behance-design](https://user-images.githubusercontent.com/59397159/251521454-d19a713c-53b8-4484-af6c-2b87e2e353c2.png)